docs/reference/ml/df-analytics/apis/put-dfanalytics.asciidoc

```diff
@@ -46,6 +46,27 @@ If the destination index already exists, then it will be use as is. This makes
 it possible to set up the destination index in advance with custom settings 
 and mappings.
 
+[[ml-put-dfanalytics-supported-fields]]
+===== Supported fields
+
+====== {oldetection-cap}
+
+{oldetection-cap} requires numeric or boolean data to analyze. The algorithms 
+don't support missing values therefore fields that have data types other than 
+numeric or boolean are ignored. Documents where included fields contain missing 
+values, null values, or an array are also ignored. Therefore the `dest` index 
+may contain documents that don't have an {olscore}. These documents are still 
+reindexed from the `source` index to the `dest`, but they are not included in 
+the {oldetection} analysis and therefore no {olscore} is computed.
+
+====== {regression-cap}
+
+{regression-cap} supports fields that are numeric, boolean, text, keyword and ip. It 
+is also tolerant of missing values. Fields that are supported are included in 
+the analysis, other fields are ignored. Documents where included fields contain 
+an array with two or more values are also ignored. Documents in the `dest` index 
```

Comment:
Do you want to re-state that `These documents are still 
reindexed from the source index to the dest, but they are not included in...` ?

Reply:
No, it's unnecessary wordiness. Thank you for spotting it.